spec/spec_helper.rb

```diff
@@ -23,6 +26,7 @@ def app
 Capybara.current_driver = :webkit
 Capybara::Webkit.configure do |config|
   config.allow_url('ajax.googleapis.com')
+  config.allow_url("fonts.googleapis.com")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.